src/release/versionHelper.d

```diff
@@ -320,40 +320,59 @@ public bool needMinorRelease ( A, B ) ( A matching_major, B matching_minor,
                                         SemVerBranch current,
                                         out Version new_version )
 {
-    import release.options;
-
-    import std.algorithm;
     import std.range;
 
     if (!matching_major.empty && matching_minor.empty)
-        with (matching_major.front)
-        {
-            int rc = 1;
+    {
+        new_version = getNextMinor(matching_major.front);
+        return true;
+    }
 
-            if (options.pre_release && prerelease.startsWith(RCPrefix))
-            {
-                import std.conv;
+    return false;
+}
 
-                rc = prerelease[RCPrefix.length .. $].to!int + 1;
-                new_version.minor = minor;
-            }
-            else if (prerelease.length == 0)
-                // Only bump minor if previous minor was not a rc, too
-                new_version.minor = minor + 1;
+/*******************************************************************************
 
-            if (options.pre_release)
-            {
-                import std.format;
-                new_version.prerelease = format("%s%s", RCPrefix, rc);
-            }
+    Find out the minor version that would follow 'last'.
+    Considers RC releases when the accordin global option was given.
 
-            new_version.major = major;
-            new_version.patch = 0;
+    Returns:
+        the next minor version to release
 
-            return true;
-        }
+*******************************************************************************/
 
-    return false;
+public Version getNextMinor ( Version last )
+{
+    import release.options;
+    import std.algorithm;
+
+    int rc = 1;
+
+    import std.stdio;
+    writefln("LAST VERSION: %s", last);
```

Comment:
I guess this slipped in and should be removed?

Reply:
ah, yes